cmd/dmsg-discovery/commands/root.go

```diff
@@ -52,6 +61,10 @@ var RootCmd = &cobra.Command{
 
 		log := sf.Logger()
 
+		if err := parse(); err != nil {
+			log.WithError(err).Fatal("No SK found.")
```

Comment:
`SecKey` may be more informative.

Reply:
Also is there a reason to have this one line function `parse`?

Reply:
You are correct, there is no need for a separate function. Will add that part directly.